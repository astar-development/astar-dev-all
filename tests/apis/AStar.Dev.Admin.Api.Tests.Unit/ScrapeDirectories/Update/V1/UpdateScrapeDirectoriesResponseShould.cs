using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Admin.Api.ScrapeDirectories.Update.V1;

[TestSubject(typeof(UpdateScrapeDirectoriesResponse))]
public sealed class UpdateScrapeDirectoriesResponseShould
{
    [Fact]
    public void ContainTheExpectedValues()
    {
        var sut = new UpdateScrapeDirectoriesResponse(new()
                                                      {
                                                          UpdatedBy           = "UpdatedBy",
                                                          UpdatedOn           = new(2001, 1, 1),
                                                          BaseDirectory       = "BaseDirectory",
                                                          BaseDirectoryFamous = "BaseDirectoryFamous",
                                                          BaseSaveDirectory   = "BaseSaveDirectory",
                                                          RootDirectory       = "RootDirectory",
                                                          SubDirectoryName    = "SubDirectoryName",
                                                          ScrapeDirectoryId   = Guid.Empty,
                                                          Id                  = 1,
                                                      });

        sut.ToJson().ShouldMatchApproved();
    }
}
