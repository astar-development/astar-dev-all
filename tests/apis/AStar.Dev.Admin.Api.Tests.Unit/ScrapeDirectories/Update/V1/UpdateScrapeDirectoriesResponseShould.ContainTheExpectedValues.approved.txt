{
  "scrapeDirectoryId": "00000000-0000-0000-0000-000000000000",
  "rootDirectory": "RootDirectory",
  "baseSaveDirectory": "BaseSaveDirectory",
  "baseDirectory": "BaseDirectory",
  "baseDirectoryFamous": "BaseDirectoryFamous",
  "subDirectoryName": "SubDirectoryName",
  "updatedBy": "UpdatedBy",
  "updatedOn": "2001-01-01T00:00:00"
}